environment/create_base_image.bash

```diff
@@ -0,0 +1,32 @@
+#! /bin/bash
+
+name=$1
+distro=$2
+
+# Make a 2048M disk
+dd if=/dev/zero of=$name.raw bs=1024 count=1 seek=2048000
+
+# Partition
+parted -s $name.raw -- mklabel msdos mkpart primary 1m 2048m toggle 1 boot
+
+# Setup loopback debice
+loopback=$(losetup --show -f $name.raw)
+partprobe $loopback
+echo "Created $loopback"
+
+# Mount first partition to /mnt
+mkfs -O ^metadata_csum -t ext4 "$loopback"p1
+mount "$loopback"p1 /mnt
+
+# Create image
+debootstrap --include=linux-image-generic $distro /mnt
```

Comment:
Did we ever have any luck with minbase and systemd? I can't remember.

May want to drop a note in here that we'd _like_ to use minbase but at least in xenial it pulls in sysvinit instead.

Reply:
No, I haven't got back at testing that. I've added the note for now